sck.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mvn -q -Pcore,cdh6.3.0 -f ./splice_ck/pom.xml exec:java -Dexec.args="$*"
```

Comment:
maybe we should be able to use this also for cdh5.12.0, or is this all the same?
```
PLATFORM=$1
shift # ignore $1
mvn -q -Pcore,${PLATFORM} -f ./splice_ck/pom.xml exec:java -Dexec.args="$*"
```

Reply:
thanks! I wrote this small script for myself. I will make it accept CDH parameter as you described.
it is not yet clear to me how we want to ship spliceck.

Reply:
I modified the script s.t. it defaults to cdh6.3.0 (since I hate typing too much :) ):

```
#!/bin/bash

platform=cdh6.3.0

function help() {
  echo "sck.sh [--platform=<platform>] command"
  echo "<platform> The cluster platform, default is cdh6.3.0"
}

if [ $# -eq 0 ]
then
  help
else
  if [ $1 = "platform" ]
  then
     platform=$2
     shift 2
  fi
  mvn -q -Pcore,${platform} -f ./splice_ck/pom.xml exec:java -Dexec.args="$*"
fi
```